src/main/java/Event.java

```diff
@@ -0,0 +1,30 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Creates an Event task with a date and time.
+ */
+public class Event extends Task {
+    protected LocalDateTime at;
+
+    /**
+     * Creates a new Event.
+     * @param description Description of the event.
+     * @param at Date and time of the event.
+     */
+    public Event(String description, LocalDateTime at) {
```

Comment:
Maybe it will be better if you name your parameter using a noun such as "date"!!